packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -65,7 +65,7 @@ class FlutterExtension {
      * Specifies the relative directory to the Flutter project directory.
      * In an app project, this is ../.. since the app's build.gradle is under android/app.
      */
-    String source
+    String source = '../..'
```

Comment:
@johnmccutchan this is actually the place where we define the `flutter {}` gradle dsl. What this change is doing is actually setting a default in our code instead of requiring that the host app (or template) set this value. 

@Gustl22 was this required for some of the other work in this pr or did you do this out of convenience? 
I have a slight preference towards not having a default value since this is the only parameter in the flutter{} block and I think having this happen by magic might lead to difficult to debug situations.